var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -15,7 +15,7 @@ class Raja(CMakePackage):
     version('develop', branch='develop', submodules='True')
     version('master',  branch='master',  submodules='True')
     version('0.11.0', tag='v0.11.0', submodules="True")
-    version('0.10.0', tag='v0.10.0', submodules="True")
+    version('0.10.1', tag='v0.10.1', submodules="True")
```

Comment:
Why is this change necessary?

Reply:
No it was a copy and paste error.  I added that version back.